india_compliance/gst_india/utils/transaction_data.py

```diff
@@ -79,7 +79,7 @@ def update_transaction_tax_details(self):
                 continue
 
             tax = self.gst_accounts[row.account_head][:-8]
-            self.transaction_details[f"total_{tax}_amount"] = abs(
+            self.transaction_details[f"total_{tax}_amount"] += abs(
```

Comment:
This would have worked correctly only for Shopify as `item_wise_tax_detail` was enforced with `dont_recompute_tax`.

Where it is auto-calculated, this would mean duplication of taxes.
eg:
![image](https://user-images.githubusercontent.com/10496564/217789697-9182c121-d8ec-4b2c-ac56-e607f6967b5d.png)